tasmota/xdrv_10_rules.ino

```diff
@@ -772,6 +772,7 @@ bool RuleSetProcess(uint8_t rule_set, String &event_saved)
       // Use Backlog with event to prevent rule event loop exception unless IF is used which uses an implicit backlog
       if ((ucommand.indexOf(F("IF ")) == -1) &&
           (ucommand.indexOf(F("EVENT ")) != -1) &&
+          (ucommand.indexOf(F("BACKLOG0 ")) == -1) &&
           (ucommand.indexOf(F("BACKLOG ")) == -1)) {
```

Comment:
Can you pls test removing your line 775 and changing line 776 into:
```
(ucommand.indexOf(F("BACKLOG")) == -1)) {
```
removing the space after "BACKLOG". This should tackle both "BACKLOG" and "BACKLOG0".